mmf/datasets/mmf_dataset_builder.py

```diff
@@ -133,17 +180,25 @@ def load(self, config, dataset_type, *args, **kwargs):
                 + "This dataset won't be used.".format(dataset_type)
             )
             return None
+        return annotations
+
+    def _calculate_split_for_dataset_type(self, dataset_type):
+        start = 0.0
+        for data_type in self.config.split_train:
+            if data_type == "seed":
+                continue
+            if dataset_type == data_type:
+                return (start, start + self.config.split_train[data_type])
+            start += self.config.split_train[data_type]
+
+        if start >= 1.0:
+            raise ValueError(
```

Comment:
This probably should be a warning in case of == 1.0 as I can launch this on val inference alone.

Reply:
added on the bottom